server/xpub-server/index.js

```diff
@@ -32,6 +35,19 @@ const registerRoutes = app => {
   require('./routes')(app)
 }
 
+const appMessage = action =>
+  `${action} Application: ${os.hostname()}, PID: ${process.pid}`
+
+const stopServer = () => {
+  // handle the application cleanup in here
+  logger.info(appMessage('Stopping'))
+}
+
+logger.info(appMessage('Starting'))
+
+// eslint-disable-next-line no-unused-vars
+const cleanupHandler = cleanup.Cleanup(process, logger, stopServer)
```

Comment:
what is `cleanupHandler` meant for?

Reply:
it exists for the lifetime of the app to route the exit event handler to the `stopServer` function